boards/common/nrf51/include/periph_conf_common.h

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
As a further improvement I think it would make sense to split this file to keep timer and RTT config in separate files. This way, boards using the shared nrf51 code can override either one independently. This would be similar to the clock config header files that we use for the shared stm32 boards. What do you think?